### PR TITLE
docs: update readme following oep-55 recommendations

### LINKED ISSUE #92: [OEP-55 PILOT] Update readme according OEP-55 specifications
### Description
The repo readme must comply with the OEP-55 specifications for phase 1 maintenance program.

### References
[Readme specification](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055/decisions/0003-readme-specification.html)

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
**Description:** 
This PR updates the readme file following the OEP-55 recommendations, especially [this](https://raw.githubusercontent.com/openedx/edx-cookiecutters/master/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/README.rst) cookie-cutter template. 

This PR resolves #92. 

**Dependencies:**
None

**Merge deadline:**
ASAP

**Installation instructions:** 
Just documentation.

**Testing instructions:**
Just documentation.

**Reviewers:**
- [x] @felipemontoya 

**Merge checklist:**
- [ ] All reviewers approved
- [x] CI build is green
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** 
~~**Question**: in the deployment section, should we add that the library works out of the box since maple?~~ already resolved

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @mariajgrimaldi!

As a core committer in this repo, you can merge this once the pull request is approved per the [core committer reviewer requirements](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Open+edX+Core+Committers#Reviews-and-Ownership) and according to the agreement with your edX Champion.


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: README.rst
Comment (mariajgrimaldi) [inline README.rst:144]: Hello @feanil! Hope you're doing great :smile: 

people without access to the github organization can access this? I asked a few colleagues, and they couldn't access it. The message error said, "user not found." Which URL should we use here?

File: README.rst
Comment (mariajgrimaldi) [inline README.rst:75]: should we add that the library works out of the box since maple?

Comment (felipemontoya) [reply README.rst:75]: Yes, we can make it clear to say that it works out of the box for the lms and cms services since it is installed by default in edx-platform.
However we can start by saying that this component is a library and therefore is not deployed but instead is added as a requirement to IDAs.

We could also add an explanation about how one could add this to an IDA that is not the edx-plat.

Comment (mariajgrimaldi) [reply README.rst:75]: That works! Thanks

**Review by felipemontoya — APPROVED** [review]
From my side this looks good now. Thanks @mariajgrimaldi

File: README.rst
Comment (e0d) [reply README.rst:144]: I assume they did login using the GitHub credentials?

Comment (mariajgrimaldi) [reply README.rst:144]: Yes! We tried logging in a new session & using a window with a session created and got the same result.

**Comment by mariajgrimaldi** [comment]
I'll merge this PR since I already raised the backstage issue in the maintainer's slack channel. For now, just folks from openedx org can access backstage.

**Comment by openedx-webhooks** [comment]
: thought you might like to know that mariajgrimaldi merged this pull request.

**Comment by openedx-webhooks** [comment]
@mariajgrimaldi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-events&entry.867055334=https://github.com/openedx/openedx-events/pull/100&entry.1484655318=https://github.com/mariajgrimaldi&entry.752974735=2022-08-30+15:26&entry.1917517419=2022-09-08+14:56&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
